pkg/edgeview/src/basics.go

```diff
@@ -350,26 +355,34 @@ func getBasics() {
 
 	if basics.server != "" {
 		var printed bool
+		var authenStr string
+		if clientAuthType != types.EvAuthTypeUnspecified {
+			authen := "certs"
+			if clientAuthType == types.EvAuthTypeSSHRsaKeys {
+				authen = "ssh-keys"
+			}
```

Comment:
Wouldn't it be more future proof to make this info a case statement with a default which generates an error and/or logs an error?
With the above approach any new value will map to "certs" if this code isn't correctly updated when the new value is added.
And that includes new values added in the JWT and not added in EvAuthType since you just do a type cast of the integer enum on line 216.